tests/integration_tests/test_examples.py

```diff
@@ -1,39 +1,48 @@
+"""This is an integration test to make sure scripts from examples/
```

Comment:
can you mark these all with `@pytest.mark.no_cover`? we don't really want to be considering a line covered just because it appears in an example.